src/main/resources/assets/js/components/Marathon.jsx

```diff
@@ -386,20 +363,20 @@ var Marathon = React.createClass({
   },
 
   render: function () {
-    var modal;
+    var page;
 
     /* jshint trailing:false, quotmark:false, newcap:false */
     /* jscs:disable disallowTrailingWhitespace, validateQuoteMarks, maximumLineLength */
-    if (this.state.modalClass === AppModalComponent) {
-      modal = (
-        <AppModalComponent
+    if (this.state.pageClass === AppPageComponent) {
+      page = (
```

Comment:
Yeah, that is done in this branch:
https://github.com/mesosphere/marathon/pull/1066